### PR TITLE
Adding TrafficManager2 SDK for .NET

### PR DESCRIPTION
**Pull request by ms-alguerra**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@t-alguer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (t-alguer). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <title>Microsoft Azure WATMv2 Management Library</title>
+    <releaseNotes>WATMv2 Service / CSM-compatible preview library</releaseNotes>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>azure-sdk, Microsoft</owners>
+    <licenseUrl>http://aka.ms/windowsazureapache2</licenseUrl>
+    <projectUrl>https://github.com/WindowsAzure/azure-sdk-for-net</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>DO NOT SHIP IN PUBLIC. MICROSOFT CONFIDENTIAL. Provides management capabilities for WATMv2 services.</summary>
```
Comment (markcowl) [inline src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec]: Please update this text

File: src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <description>DO NOT SHIP IN PUBLIC. MICROSOFT CONFIDENTIAL.
```
Comment (markcowl) [inline src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec]: same

File: src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    <licenseUrl>http://aka.ms/windowsazureapache2</licenseUrl>
+    <projectUrl>https://github.com/WindowsAzure/azure-sdk-for-net</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>DO NOT SHIP IN PUBLIC. MICROSOFT CONFIDENTIAL. Provides management capabilities for WATMv2 services.</summary>
+    <description>DO NOT SHIP IN PUBLIC. MICROSOFT CONFIDENTIAL. 
+CSM/Resource Groups-compatible library.
+Provides management capabilities for WATMv2 services.</description>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <tags>Microsoft Azure WATMv2 management "WATMv2" "WATMv2 management" REST HTTP client windowsazureofficial</tags>
```
Comment (markcowl) [inline src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec]: Do you want to refer to WATMV2 in the title or tags?

File: src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec [hunk 4]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata minClientVersion="2.5">
+    <id>Microsoft.Azure.Management.TrafficManager</id>
+    <title>Microsoft Azure WATMv2 Management Library</title>
```
Comment (markcowl) [inline src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec]: Is this the correct title?

File: src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec (hunk 1 above)
Comment (t-alguer) [inline src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec]: What do you want me to update?

File: src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec (hunk 2 above)
Comment (t-alguer) [inline src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec]: It is the same file, right?

File: src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec (hunk 3 above)
Comment (t-alguer) [inline src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec]: I changed our code word for the service name. Let me know if that is OK. Should I refer to WATM? What is the difference between doing it in the title or tags?

File: src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec (hunk 4 above)
Comment (t-alguer) [inline src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec]: Changed

File: src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    <title>Microsoft Azure Traffic Manager Management Library</title>
+    <releaseNotes>Traffic Manager Service / CSM-compatible preview library</releaseNotes>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>azure-sdk, Microsoft</owners>
+    <licenseUrl>http://aka.ms/windowsazureapache2</licenseUrl>
+    <projectUrl>https://github.com/Azure/azure-sdk-for-net</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>DO NOT SHIP IN PUBLIC. MICROSOFT CONFIDENTIAL. Provides management capabilities for Traffic Manager services.</summary>
```
Comment (markcowl) [inline src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec]: fix

File: src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    <description>DO NOT SHIP IN PUBLIC. MICROSOFT CONFIDENTIAL.
```
Comment (markcowl) [inline src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec]: fix

**Comment by t-alguer** [comment]
@markcowl Take a look at it now. It should be fine.

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by yugangw-msft** [comment]
close this one since https://github.com/Azure/azure-sdk-for-net/pull/1100 should include the same change

**Pull request closed without merging**